core/templates/pages/exploration-editor-page/editor-tab/services/solution-verification.service.spec.ts

```diff
@@ -21,33 +21,34 @@
 import { UpgradedServices } from 'services/UpgradedServices';
 // ^^^ This block is to be removed.
 
-import { AngularNameService } from
-  'pages/exploration-editor-page/services/angular-name.service';
-
-require('App.ts');
-require('domain/exploration/SolutionObjectFactory.ts');
+import { TestBed } from '@angular/core/testing';
+import { SolutionObjectFactory } from 'domain/exploration/SolutionObjectFactory.ts';
 require('pages/exploration-editor-page/services/exploration-states.service.ts');
-require(
-  'pages/exploration-editor-page/editor-tab/services/' +
-  'solution-verification.service.ts');
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-customization-args.service.ts');
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-editor.service.ts');
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-interaction-id.service.ts');
+import { SolutionVerificationService } from
+  // eslint-disable-next-line max-len
+  'pages/exploration-editor-page/editor-tab/services/solution-verification.service.ts';
```

Comment:
import statements are exempted from max-len violations. Can you change that here and everywhere else. No need to break them into two lines.
```suggestion
import { SolutionVerificationService } from 'pages/exploration-editor-page/editor-tab/services/solution-verification.service.ts';
```